templates/todo/projects/nodejs-mongo-aks/.repo/bicep/azure.yaml

```diff
@@ -28,7 +43,7 @@ services:
       postdeploy:
         windows:
           shell: pwsh
-          run: azd env set REACT_APP_API_BASE_URL $env:SERVICE_API_ENDPOINT_URL
+          run: azd env set API_BASE_URL $env:SERVICE_API_ENDPOINT_URL
         posix:
           shell: sh
-          run: azd env set REACT_APP_API_BASE_URL ${SERVICE_API_ENDPOINT_URL}
\ No newline at end of file
+          run: azd env set API_BASE_URL ${SERVICE_API_ENDPOINT_URL}
```

Comment:
we don't need to do this.  Use the `SERVICE_API_ENDPOINT_URL` env var directly when creating the .env.local file for the web. See: https://github.com/Azure/azure-dev/pull/4117